### ISSUE TITLE
own data set training issue

### ISSUE THREAD

**Issue by usu008** [tensorflow/models#3043]
whene iam start training my own data set it shows the following error...i already reduce the batch size and queue again it shows the error like this

any one plz help me to solve this

INFO:tensorflow:global step 470: loss = 0.8664 (5.802 sec/step)
INFO:tensorflow:global step 471: loss = 0.8947 (8.888 sec/step)
INFO:tensorflow:global step 472: loss = 1.9891 (6.110 sec/step)
INFO:tensorflow:global step 473: loss = 1.4290 (3.640 sec/step)
INFO:tensorflow:global step 474: loss = 0.5736 (13.695 sec/step)
INFO:tensorflow:global step 475: loss = 0.7079 (4.174 sec/step)
INFO:tensorflow:global step 476: loss = 0.8392 (13.596 sec/step)
INFO:tensorflow:global step 477: loss = 0.6483 (2.579 sec/step)
INFO:tensorflow:global step 478: loss = 0.5341 (10.204 sec/step)
INFO:tensorflow:global step 479: loss = 0.7850 (22.350 sec/step)
INFO:tensorflow:global step 480: loss = 1.9921 (3.532 sec/step)
INFO:tensorflow:global step 481: loss = 0.8216 (5.998 sec/step)
INFO:tensorflow:Recording summary at step 481.
Killed

**Comment by k-w-w** [comment]
Which model are you running? Did it say why the program was killed?

**Comment by jart** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by usu008** [comment]
its didn't say anything about the abnormal termination....but  i think it may be occure because of my image resolution .....here iam using "generate_tfrecord.py" file to generate the record file.....
i found some solution from stackoverflow  they said instead of that if i use "create_pet_tf_record" means it will speed up my train process without any termination......
so...any one know how to use   "create_pet_tf_record"?
